app/assets/javascripts/models/ui/reverb_retirement.coffee

```diff
@@ -16,14 +17,32 @@ ns.ReverbRetirement = do (ko) ->
 
     returnToReverb: (source = 'modal link') =>
       Cookies.set('ReadyForReverbRetirement', 'false', { expires: 90 })
-      # metrics go here once it's figured out...
-      # metrics_event('reverb_redirect', 'back_to_reverb', {source: source}) 
+
+      data['type'] = 'reverb_redirect'
+      data['data'] = 'back_to_reverb'
+      data['other_data'] = source
+      ajax
+        data: JSON.stringify(data)
+        dataType: 'json'
+        url: "/metrics"
+        method: 'post'
+        success: (data) ->
+          console.log data
       window.location.replace("https://" + document.referrer.match(/:\/\/(.[^/]+)/)[1])
     
     stayWithEDSC: () =>
       Cookies.set('ReadyForReverbRetirement', 'true', { expires: 90 })
-      # metrics go here once it's figured out...
-      # metrics_event('reverb_redirect', 'stay_in_edsc') %>
+
+      data['type'] = 'reverb_redirect'
+      data['data'] = 'stay_in_edsc'
+      data['other_data'] = ""
```

Comment:
Add source here?

Reply:
This one doesn't need it - it's the "stay in edsc" choice which has only a single source (the modal).  It's only the routes back to reverb (toolbar versus modal) that need a source.